code/modules/admin/topic.dm

```diff
@@ -955,11 +970,13 @@
 						continue
 					joblist += jobPos
 			if("ghostroles")
-				joblist += list(ROLE_PAI, "posibrain", "drone", "deathsquad", "lavaland")
+				joblist += list(ROLE_PAI, "posibrain", "drone", "deathsquad", "lavaland", ROLE_SENTIENCE)

```

Comment:
convert `"posibrain", "drone", "deathsquad", "lavaland"` into defines while you're here.